enigma-server/src/main/java/org/quiltmc/enigma/network/packet/PacketHelper.java

```diff
@@ -145,27 +146,43 @@ public static EntryChange<?> readEntryChange(DataInput input) throws IOException
 		EntryChange<?> change = EntryChange.modify(e);
 
 		int flags = input.readUnsignedByte();
-		TristateChange.Type deobfNameT = TristateChange.Type.values()[flags & 0x3];
-		TristateChange.Type javadocT = TristateChange.Type.values()[flags >> 2 & 0x3];
+		TristateChange.Type deobfNameType = TristateChange.Type.values()[flags & 0x3];
+		TristateChange.Type javadocType = TristateChange.Type.values()[flags >> 2 & 0x3];
+		TristateChange.Type tokenTypeType = TristateChange.Type.values()[flags >> 2 & 0x3];
+		TristateChange.Type pluginIdType = TristateChange.Type.values()[flags >> 2 & 0x3];
```

Comment:
Is it intentional that these have the same value as `javadocType`?

Reply:
nope that's broken, good catch!